src/appleseedmaya/exporters/shadingnodeexporter.cpp

```diff
@@ -58,6 +58,12 @@ namespace asr = renderer;
 namespace
 {
 
+const char *g_compToColorParamNames[] = {"compR", "compG", "compB"};
+const char *g_compToColorOutputParamName = "comp";
+
+const char *g_colorToCompParamNames[] = {"compR", "compG", "compB"};
```

Comment:
My point is that I see two C arrays with the exact same values. It seems redundant.